samples/meetings-share-to-stage-signing/csharp/README.md

```diff
@@ -13,13 +13,30 @@ createdDate: "09-03-2022 15:56:00"
 ---
 
 # In-Meeting Document Signing
-This proof of concept demonstrates the use case of sharing a document on Teams Share to Stage with live signing feature in a meeting.
+Meeting Signing is a Teams sample app that demonstrates the scenario of enabling users to sign documents in real time during meetings making possible to review and sign documents in a single session. This sample will let users electronically sign documents by certifying their identity. Many kind of documents like NDAs, Purchase Orders and Contracts can be signed during meetings where real time collaboration happens. This template can be used for multiple scenarios such as approve PRs, Purchase Orders, Document Changes 
+
+The sample app can be customized to incorporate other kind of signatures like manual signatures, Tax Offices Certified signatures among others. 
```

Comment:
I'm not sure what you mean by this? what is a manual signature or tax office certified signatures?

Reply:
There are other kind of signatures that Signature ISVs offer (manual signature, business signatures)